presto-docs/src/main/sphinx/release/release-0.216.rst

```diff
@@ -0,0 +1,65 @@
+=============
+Release 0.216
+=============
+
+General Changes
+---------------
+
+* Fix incorrect query result for queries involving timestamp, when the time zone database
+  embedded in the version of Joda-Time that Presto depends on does not match the time zone
+  database used by the JVM that Presto runs on.
+* Fix count(*) aggregation on empty relation when ``optimize_mixed_distinct_aggregation`` is enabled.
+* Improve performance of table scan on structured types by removing unnecessary memory accounting.
+* Add `reduce_agg` aggregate function.
+* Add :func:`millisecond` function (:issue:`#9524`).
+* Add an optimizer rule to push filters on window function partition symbols under the window
```

Comment:
This release note item is talking in terms of the plan tree. Can we make it more user friendly? 
We should say something like `Add an optimizer rule to filter the window partitions before executing the window operators`. Please check with the author of this change if this statement is correct.

Reply:
@sopel39 Karol, could you review the proposed wording?

Reply:
Much better, thanks!